plugins/woocommerce/client/legacy/js/frontend/woocommerce.js

```diff
@@ -182,9 +182,27 @@ jQuery( function ( $ ) {
 		} );
 	} );
 
+	// If the "Enable AJAX add to cart buttons on archives" setting is disabled
+	// the add-to-cart.js file won't be loaded, so we need to add the event listener here.
+	if ( typeof wc_add_to_cart_params === 'undefined') {
+		$( document.body ).on( 'keydown', '.remove_from_cart_button', on_keydown_remove_from_cart );
+	}
+
 	$( document.body ).on( 'item_removed_from_classic_cart', focus_populate_live_region );
 } );
 
+/**
+ * Handle when pressing the Space key on the remove item link.
+ * This is necessary because the link has the role="button" attribute
+ * and needs to act like a button.
+ */
+function on_keydown_remove_from_cart( e ) {
+	if ( e.key === ' ' ) {
+		e.preventDefault();
+		e.currentTarget.click();
+	}
+}
```

Comment:
Finally, in this case, we could also use `event` over `e`:

```suggestion
function on_keydown_remove_from_cart( event ) {
	if ( event.key === ' ' ) {
		event.preventDefault();
		event.currentTarget.click();
	}
}
```